src/einsteinpy/hypersurface/embedding.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+from mpl_toolkits import mplot3d
+from astropy import units as u
+
+"""
+Class for schwarzschild embedding hypersurface functionalities
+"""
+
+class Embedding:
```

Comment:
Class name must be more descriptive

Reply:
Class name should indicate that the hypersurface is specific to `Schwarzschild` space-time. It looks generic. Maybe `SchwarzschildEmbedding`. Suggestions @shreyasbapat ?